### PR TITLE
Filter component updated

### PR DESCRIPTION
**Pull request by Smit2010**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
     "formik": "^2.1.4",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-redux": "^7.1.3",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.0",
+    "react-select": "^3.0.8",
     "react-toastify": "^5.5.0",
     "redux": "^4.0.5",
+    "redux-form": "^8.3.1",
```
Comment (pranavpandey1998official) [inline package.json]: we are already using formik

File: package.json [hunk 2]
```diff
@@ -3,18 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.9.5",
```
Comment (pranavpandey1998official) [inline package.json]: bhai bulma ke saath karle

Comment (Smit2010) [reply package.json]: Arre uske UI components are not as good as material UI

Comment (Smit2010) [reply package.json]: And bulma does not provide slider.

**Pull request closed without merging**